test/Input.test.jsx

```diff
@@ -0,0 +1,33 @@
+import { fireEvent, render } from '@testing-library/react';
+import Input from '../src/Input';
+
+describe('Input', () => {
+  test('초기 렌더링 시 "할 일" 과 "추가", value props 가 노출된다.', () => {
+    const handleChange = jest.fn();
+    const { container, getByRole } = render(<Input value="hello" onChange={handleChange} />);
```

Comment:
```suggestion
    const { container, getByRole } = render((
      <Input
        value="hello"
        onChange={handleChange}
      />
    ));        
```

저는 props가 2개 이상일 때는 줄바꿈해주면 코드가독성이 더 좋다고 생각하고 코드를 정리해주는 편이에요.
코드 가독성을 고려한 자기만의 코드 정리 기준을 세우시면 좋아요.

Reply:
넵넵!! 좋은 것 같습니다!